sdk/bento/graph/ast.py

```diff
@@ -73,10 +90,11 @@ def load_ast_module(ast: AST) -> Any:
     src = unparse(ast)
     with NamedTemporaryFile(mode="w", suffix=".py", delete=False) as f:
         f.write(src)
-        f.flush()
+        f.close()
         # import the source as a module
         mod_spec = spec_from_file_location("compiled", f.name)
         module = module_from_spec(mod_spec)
         mod_spec.loader.exec_module(module)
+    os.remove(f.name)
```

Comment:
You passed in `delete=False` that's why the file is not removed. No need to `os.remove` the file.

Reply:
Auto deletion for `NamedTemporaryFile` is [broken](https://github.com/bravoserver/bravo/issues/111) on Windows for some reason

Reply:
That's really weird, I would expect the fix to be something simple, like when the context manager exits, it also calls `os.remove`. I guess you can just document it in the code

Reply:
Documented.